doc/tools/make_rst.py

```diff
@@ -61,22 +59,22 @@
     "This method doesn't need an instance to be called, so it can be called directly using the class name.",
     "This method describes a valid operator to use with this type as left-hand operand.",
 ]
-strings_l10n = {}
+strings_l10n: Dict[str, str] = {}
 
-STYLES = {}
+STYLES: Dict[str, str] = {}
 
 
-def print_error(error, state):  # type: (str, State) -> None
+def print_error(error: str, state: "State") -> None:
     print("{}{}ERROR:{} {}{}".format(STYLES["red"], STYLES["bold"], STYLES["regular"], error, STYLES["reset"]))
     state.num_errors += 1
 
 
 class TypeName:
-    def __init__(self, type_name, enum=None):  # type: (str, Optional[str]) -> None
+    def __init__(self, type_name: str, enum: Optional[str] = None) -> None:
         self.type_name = type_name
         self.enum = enum
 
-    def to_rst(self, state):  # type: ("State") -> str
+    def to_rst(self, state: "State") -> str:
```

Comment:
Same question here for the stringly typing.

Reply:
Yes & can move it.